W-17118609 doc updates for Nov GA jh (#55)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/limitations.adoc

* API Experience Hub functions at the main organization level only.
* A Salesforce organization can't connect to more than one Anypoint organization.
* Multiple portals per Salesforce organization aren’t supported.
* API Groups aren't available.
* An API portal can have a maximum of 350 APIs.
+
Don't make changes to the permissions of the out-of-the-box Teams created in Access Management for API Experience Hub. Each asset added to the API portal uses two permissions and Access Management has a limit of 1000 permissions per team.

> * If scalability is enabled on the *API management* page, an API portal can have a maximum of 1,000 APIs. After this option is enabled, you can't revert to the portal version that supports up to 350 APIs.

Reviewer: ```suggestion
* Enable scalability on the *API management* page to increase the maximum number of APIs to 1,000 for the API portal. After you enable this option, you can't revert to the portal version that supports up to 350 APIs.
```

> * If scalability is enabled on the *API management* page, an API portal can have a maximum of 1,000 APIs. Once this option is enabled, you can't revert to the portal version supporting the maximum of 350 APIs.

Reviewer 2: Should management be init cap?

Author: Yes, the UI uses sentence case

> * If scalability is enabled on the *API management* page, an API portal can have a maximum of 1,000 APIs. Once this option is enabled, you can't revert to the portal version supporting the maximum of 350 APIs.

Reviewer 2: ```suggestion
* If scalability is enabled on the *API management* page, an API portal can have a maximum of 1,000 APIs. After this option is enabled, you can't revert to the portal version that supports up to 350 APIs.
```

Reviewer 2: Don't use "[once](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22once%22)" in this way.

+ 

> For more information, see xref:managing-apis.adoc#opt-in-to-increase-the-number-of-apis[Opt In to Increase the Number of APIs In the Portal]. 

Reviewer 2: ```suggestion
For more information, see xref:managing-apis.adoc#opt-in-to-increase-the-number-of-apis[Opt in to Increase the Number of APIs in the Portal]. 
```
